suite-native/storage/src/typedPersistReducer.ts

```diff
@@ -8,13 +8,23 @@ export const preparePersistReducer = async <TReducerInitialState>({
     persistedKeys,
     key,
     version,
+    migrations,
 }: {
     reducer: Reducer<TReducerInitialState>;
     persistedKeys: Array<keyof TReducerInitialState>;
     key: string;
     version: number;
-}) =>
-    persistReducer(
-        { key, storage: await initMmkvStorage(), whitelist: persistedKeys as string[], version },
-        reducer,
-    );
+    migrations?: { [key: string]: (state: any) => any };
+}) => {
+    const migrate = createMigrate(migrations ?? {}, { debug: false });
```

Comment:
what is `debug: false`?

Reply:
It is just disabling logging during migration. We can turn this on however I'm not sure if that's helpful as it will be logging into console I guess